File: src/main/java/me/jsinco/oneannouncer/discord/DiscordSRVUtil.java
package me.jsinco.oneannouncer.discord;

import github.scarsz.discordsrv.DiscordSRV;

import java.util.UUID;

public class DiscordSRVUtil {

    DiscordSRV discordSRV = DiscordSRV.getPlugin();


    public String getLinkedAccountID(String minecraftUUID) {
        return discordSRV.getAccountLinkManager().getDiscordId(UUID.fromString(minecraftUUID));
    }


    public boolean sendMessageFromMinecraft(String minecraftUUID, String msg) {
        String discordID = getLinkedAccountID(minecraftUUID);
        if (discordID != null) {
            JDAMethods.sendMessageDiscordUser(discordID, msg);
        }
        return discordID != null;
    }
}

File: src/main/java/me/jsinco/oneannouncer/DiscordSRVUtil.kt
package me.jsinco.oneannouncer

import github.scarsz.discordsrv.DiscordSRV
import java.util.UUID

object DiscordSRVUtil {

    fun getDiscordIDFromUUID(uuid: UUID): String? {
        return DiscordSRV.getPlugin().accountLinkManager.getDiscordId(uuid)
    }

    fun getUUIDFromDiscordID(discordID: String): UUID? {
        return DiscordSRV.getPlugin().accountLinkManager.getUuid(discordID)
    }
}
